examples/appprotect/README.md

```diff
@@ -25,17 +25,30 @@ $ kubectl create -f cafe.yaml
 ```
 
 ## 3. Configure Load Balancing
-
-1. Create a secret with an SSL certificate and a key:
+1. Create the syslog service and pod for the App Protect security logs:
+    ```
+    $ kubectl create -f syslog.yaml
+    ```
+2. Create a secret with an SSL certificate and a key:
     ```
     $ kubectl create -f cafe-secret.yaml
     ```
-2. Create the App Protect policy and log configuration:
+3. Create the App Protect policy and log configuration:
+    ```
+    $ kubectl create -f dataguard-alarm.yaml
+    $ kubectl create -f logconf.yaml
+    ```
+4. Create an Ingress resource:
+    Get the previously deployed syslog IP:
     ```
-    kubectl create -f dataguard-alarm.yaml
-    kubectl create -f logconf.yaml
+    $ kubectl get ep syslog-svc
```

Comment:
hm, just curious why not to use the VIP of the service?

Reply:
I remember someone told me it wouldn't work, but I'll double check

Reply:
It seems we have to use TCP for this example